jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/Lambda.kt

```diff
@@ -78,4 +79,26 @@ fun UpdateFunctionConfigurationResponse.toDataClass(credentialProviderId: String
     role = IamRole(this.role()),
     credentialProviderId = credentialProviderId,
     region = region
-)
\ No newline at end of file
+)
+
+fun GetFunctionConfigurationResponse.toConfiguration() = FunctionConfiguration.builder()
```

Comment:
We end up just coveting this onto a LambdaFunction in the init of the Lambda node, should we refactor LambdaNode to take a LambdaFunction and save us this middle conversion?

Reply:
👍